CHANGELOG.md

```diff
@@ -7,7 +7,9 @@ and [Keep a Changelog](http://keepachangelog.com/).
 
 ## Upcoming version
 
-No changes yet.
+### Changed
+
+- Applied code formatting rules from [PSR](https://www.php-fig.org/psr/) 2 and 12 ([#142])
```

Comment:
Considering that [PSR-2](https://www.php-fig.org/psr/psr-2/) was deprecated in favor of [PSR-12](https://www.php-fig.org/psr/psr-12/) (which appears to be a superset of the former), shouldn't we mention only the latter?

Reply:
Fair point, I'll hitch-hike that into #143.